cves/CVE-2012-6617.yml

```diff
@@ -344,7 +342,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: I believe a lack of testing is the cause of this mistake. I think if they tested with a wider range of vectors, this bug could have been caught.
```

Comment:
Similar to what I put in the other file - Do you think they might not have known better at the time?. I looked at the vcc real quick and it was made back in 2007

Reply:
Is there a way to find out what the standard was back then? I don't want to assume that even though it was over a decade ago, this information wasn't out there and they couldn't have known.

Reply:
I think they just didn't think of it. Security should always be the first thing you think about.